### ISSUE TITLE
Support query APIs in SAI Player

### ISSUE THREAD

**Issue by daall** [Azure/sonic-sairedis#527]
SAI Player should be able to play back query APIs that have been added like sai_query_attribute_enum_values_capability and sai_object_type_get_availability.

**Comment by kcudnik** [comment]
yes, but they are not critical, this is just simple GET api that don't return any oid values so from replay perspective is useless (except performance time)
